package.json

```diff
@@ -64,7 +64,7 @@
   "readmeFilename": "README.md",
   "peerDependencies": {
     "react": ">= 16.6.3",
-    "react-native": ">= 0.58.2"
+    "react-native": "*"
```

Comment:
are these still needed?

Reply:
yes this is needed so expo installs the dependency correctly without having to force it. 0.58.2 is pretty old so we should be fine. react-native-purchases-ui also has *